sklearn/model_selection/_split.py

```diff
@@ -574,6 +576,8 @@ class StratifiedKFold(_BaseKFold):
 
     Read more in the :ref:`User Guide <cross_validation>`.
 
+    .. versionadded:: 0.13
```

Comment:
depends how you account for the transition from sklearn.cross_validation to sklearn.model_selection...